### PR TITLE
Rework overview/what are extensions page

### PR DESCRIPTION
**Pull request by awfuchs**
Update and clean up the page at https://developer.chrome.com/docs/extensions/mv3/overview/. Initially, remove the "Featured videos" section that points to videos 10+ years old. Then:

* Do some general tidying and polishing
* Make other updates based on stakeholder input

### CODE REVIEW

File: site/en/docs/extensions/mv3/overview/index.md [hunk 1]
```diff
@@ -6,9 +6,24 @@ updated: 2018-02-25
...
-technologies such as HTML, JavaScript, and CSS.
+This page provides a very brief introduction to what extensions are and how you can make a trivial one.
+
+
+## About extensions
+
+Extensions are small software programs that customize the browsing experience. They let users
+tailor Chrome functionality and behavior in many ways, providing things like:
+
+* Games
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/overview/index.md:17]: I'm very hesitant about having "games" be the first example of what extensions can do. Honestly I'm not even sure if we should include it in this list. 

Broadly speaking extensions are meant as a way to extend your web browser's functionality. You might want to customize how a feature works (e.g. replace the browser's bookmark manager), add new functionality (e.g. look up a word by highlighting it), integrate tools that keep you on track (e.g. blocking distracting websites), and so on.

While there are some games in the Chrome Web Store, I don't think this is the right focus for the extensions platform. Generally speaking I think games are better as web sites rather tightly integrated into the browser experience.

File: site/en/docs/extensions/mv3/overview/index.md [hunk 2]
```diff
@@ -20,7 +35,9 @@ contributes towards a common purpose.
 User interfaces should be minimal and have intent. They can range from a simple icon, such as the [Google
 Mail Checker extension][2] shown on the right, to [overriding][3] an entire page.
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/overview/index.md:36]:
```suggestion
User interfaces should be minimal and have intent. They can range from a simple icon, such as the [Google
Mail Checker extension][2] shown above, to [overriding][3] an entire page.
```

Also, can we update the screenshot of the Google Mail checker image? Chrome's UI has changed since this image was captured.

File: site/en/docs/extensions/mv3/overview/index.md [hunk 3]
```diff
@@ -20,7 +35,9 @@ contributes towards a common purpose.
...
 
-Extension files are zipped into a single `.crx` package that the user downloads and installs. This
+### How do users get extensions?
+
+When you make an extension, you zip it into a single `.crx` package that the user downloads and installs. This
 means extensions do not depend on content from the web, unlike ordinary web apps.
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/overview/index.md:41]: I suspect that the original version used the passive voice because CWS is typically the entity that generates the crx package that a user downloads.

I think it may be better to pivot this section (+1 to creating a new heading) a little to focus more on CWS. While we may want to talk about what a CRX file is, I'm not sure it fits under this heading. To that end here's a quick attempt to draft up a new body for this section.

> Most Chrome users get extensions from the Chrome Web Store (CWS). Developers across the globe submit their extension to the Chrome Web Store where they are reviewed and made available to end users. 
> 
> Some businesses and schools use [enterprise policies](https://cloud.google.com/docs/chrome-enterprise/policies) to install extensions on their user's devices. These extensions are either fetched from the Chrome Web Store or they may be hosted on web servers. See XXX for info on the capabilities and limitations of  self hosting.

File: site/en/docs/extensions/mv3/overview/index.md [hunk 4]
```diff
@@ -114,10 +131,6 @@ or by pressing `Ctrl+Shift+F` on your keyboard.
 3.  Keep up to date by reading the [Chromium blog][12]
 4.  Subscribe to the [chromium-extensions group][13]
 
-## Featured videos {: #Featured-videos }
-
-[Technical videos][14]
-[Developer snapshots][15]
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/overview/index.md:120]: +1 to dropping this.

Much broader topic, but later in the year (Q2 maybe?) we should consider what video content might be worth producing.

**Review by dotproto — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/mv3/overview/index.md (hunk 1 above)
Comment (samthor) [reply site/en/docs/extensions/mv3/overview/index.md:17]: Could we say "Delightful experiences" aka something like "Fun", instead of "Games"?

This would describe to me [this extension](https://chrome.google.com/webstore/detail/tabby-cat/mefhakmgclhhfbdadeojlkbllmecialg?hl=en).

Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md]: Changed this to "Fun & games" as the last bullet.

File: site/en/docs/extensions/mv3/overview/index.md (hunk 3 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md]: Adapted the suggested text to replace the content of this section.

File: site/en/docs/extensions/mv3/overview/index.md [hunk 5]
```diff
@@ -6,25 +6,79 @@ updated: 2018-02-25
...
+sandboxed execution environment and interact with the Chrome browser.  Consider the context for this
+-- Chrome runs on a device, connecting the user to resources on the Web, as depicted in this context
+diagram:
+
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/5qVlqGn6vVMCWCSRclOY.png", alt="Context diagram of Chrome with an extension", width="517", height="476" %}
+
+Also shown in the diagram is an extension "plugged into" the Chrome browser. The extension can
+access and modify selected resources and actions in Chrome. There are two aspects to consider:
+
+**The extensions mechanism**&emsp;This provides a consistent way for users to install and activate
```
Comment (rdcronin) [inline site/en/docs/extensions/mv3/overview/index.md:37]: I don't love the phrasing of this as "the extensions mechanism" - it's not clear to me what this is referring to, or what the bounds of it are.  If this is just the installation and activation, it's more the UI than the "mechanism".  Can you add a bit more detail here about what this is trying to get at?

File: site/en/docs/extensions/mv3/overview/index.md [hunk 6]
```diff
@@ -6,25 +6,79 @@ updated: 2018-02-25
...
+extensions.
+
+**Extensions APIs**&emsp;The [extensions APIs](/docs/extensions/reference/) allow the extension's
+code to access features of the browser itself, activating tabs, modifying net requests, and so on.
```
Comment (rdcronin) [inline site/en/docs/extensions/mv3/overview/index.md:41]:
```suggestion
code to access features of the browser itself: activating tabs, modifying net requests, and so on.
```

(or an em dash, or insert a "such as")

File: site/en/docs/extensions/mv3/overview/index.md [hunk 7]
```diff
@@ -6,25 +6,79 @@ updated: 2018-02-25
...
+
+To create an extension, you assemble some resources -- JavaScript and HTML files, a manifest,
```
Comment (rdcronin) [inline site/en/docs/extensions/mv3/overview/index.md:43]: nit: maybe lead with "manifest", since it's the only required bit?

File: site/en/docs/extensions/mv3/overview/index.md [hunk 8]
```diff
@@ -6,25 +6,79 @@ updated: 2018-02-25
...
+{# TODO: See XXX for info on the capabilities and limitations of self hosting. #}
+
+{% endAside %}
+
+You can distribute your extensions through the [Chrome Developer Dashboard][4], publishing them to the [Chrome
+Web Store][5]. For more information, see the Chrome Web Store [developer documentation][6].
+
+### A note about extensions policy
+
+Extensions on the Chrome Web Store must adhere to the [Chrome Web Store policies][cws-policies].
```
Comment (rdcronin) [inline site/en/docs/extensions/mv3/overview/index.md:67]: nit: Can we use a consistent link style?  (I'd lean towards [named-style] instead of [numbered-style], since it's much more clear IMO)

File: site/en/docs/extensions/mv3/overview/index.md [hunk 9]
```diff
@@ -6,25 +6,79 @@ updated: 2018-02-25
...
 
-{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/bk1NEBu7WXWCXfNJCsms.png",
-       alt="A screenshot of an extension's icon in the browser bar", height="35", width="91" %}
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/XniXB3snAeMvLwI1am3O.png", alt="Screenshot of AMP validator extension pinned", width="169", height="62" %}
 
-User interfaces should be minimal and have intent. They can range from a simple icon, such as the [Google
-Mail Checker extension][2] shown on the right, to [overriding][3] an entire page.
+* User interfaces should be minimal and have intent. They can range from a simple icon, such as the
+  [AMP validator] extension shown above, to opening a new window with a form, like the [PackageTrack]
+  extension shown below.
```
Comment (rdcronin) [inline site/en/docs/extensions/mv3/overview/index.md:78]: Can we use a google-published extension, rather than PackageTrack?

File: site/en/docs/extensions/mv3/overview/index.md [hunk 10]
```diff
@@ -110,19 +164,17 @@ or by pressing `Ctrl+Shift+F` on your keyboard.
 ## What next? {: #How-do-I-start }
 
 1.  Follow the [Getting Started tutorial][10]
-2.  Read the [Overview][11]
-3.  Keep up to date by reading the [Chromium blog][12]
-4.  Subscribe to the [chromium-extensions group][13]
-
-## Featured videos {: #Featured-videos }
+1.  Explore the [extension samples][2]
+1.  Subscribe to the [chromium-extensions group][13]
```
Comment (rdcronin) [inline site/en/docs/extensions/mv3/overview/index.md:168]:
```suggestion
2.  Subscribe to the [chromium-extensions group][13]
```

File: site/en/docs/extensions/mv3/overview/index.md [hunk 11]
```diff
@@ -6,25 +6,79 @@ updated: 2018-02-25
...
+Store][5] to see thousands of different examples of published extensions.
+
+### How do extensions work?
+
+Extensions are built on web technologies such as HTML, JavaScript, and CSS.  They run in a separate,
+sandboxed execution environment and interact with the Chrome browser.  Consider the context for this
+-- Chrome runs on a device, connecting the user to resources on the Web, as depicted in this context
+diagram:
+
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/5qVlqGn6vVMCWCSRclOY.png", alt="Context diagram of Chrome with an extension", width="517", height="476" %}
```
Comment (rdcronin) [inline site/en/docs/extensions/mv3/overview/index.md:32]: I'm not sure I understand this diagram.  Is the extension a part of Chrome?  Does it not have access to HTML APIs, storage, and UI (which are separate)?  Can it not access the internet?

Can you describe what the main takeaway we want the reader to have from this diagram is?

**Review by rdcronin — CHANGES_REQUESTED** [review]
Thanks, Andy!  This is a great intro.

File: site/en/docs/extensions/mv3/overview/index.md (hunk 8 above)
Comment (dotproto) [reply site/en/docs/extensions/mv3/overview/index.md:67]: The main reason we currently use numbers is that our automated tooling assigned numbers when we ported content over. I don't have any objection to normalizing on named-style. If we do that I'd favor sorting links in the footer alphabetically.

File: site/en/docs/extensions/mv3/overview/index.md (hunk 9 above)
Comment (dotproto) [reply site/en/docs/extensions/mv3/overview/index.md:78]: +1. If we do want to use 3rd party extensions we should probably discuss offline what steps are required to do so.

File: site/en/docs/extensions/mv3/overview/index.md (hunk 8 above)
Comment (rdcronin) [reply site/en/docs/extensions/mv3/overview/index.md:67]: Yep, and I don't think we need to go on a spree updating each individual page.  But if we're heavily modifying this one already, I think we should fix it up while we're here. : )

File: site/en/docs/extensions/mv3/overview/index.md (hunk 11 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md:32]: What I'd like to depict is that Chrome does its thing as a browser, and then extensions "plug onto" Chrome and have access (via the API) of selected Chrome features to intercede in the way they work.

File: site/en/docs/extensions/mv3/overview/index.md (hunk 10 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md:168]: It's better Markdown practice to make numbered lists go 1., 1., 1., ... and let the processor handle the numbering. Check the staging preview.

File: site/en/docs/extensions/mv3/overview/index.md (hunk 9 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md:78]: @dotproto Any particular recommendation?

File: site/en/docs/extensions/mv3/overview/index.md (hunk 11 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md:32]: For things like this I miss whiteboarding in the office...

File: site/en/docs/extensions/mv3/overview/index.md (hunk 5 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md]: Done: Changed it to "extensions user interface" and made it about "manage their extensions"

File: site/en/docs/extensions/mv3/overview/index.md (hunk 7 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md]: Done

File: site/en/docs/extensions/mv3/overview/index.md (hunk 8 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md:67]: And I often like [inline-style](...) ;-)

I have updated the links in this doc to be all [named-style] and alphabetized the definitions part.

File: site/en/docs/extensions/mv3/overview/index.md (hunk 10 above)
Comment (rdcronin) [reply site/en/docs/extensions/mv3/overview/index.md]: Nifty!  I saw that the rendering was correct, but didn't realize this was a feature of markdown.  Good to know.

File: site/en/docs/extensions/mv3/overview/index.md (hunk 11 above)
Comment (rdcronin) [reply site/en/docs/extensions/mv3/overview/index.md:32]: Hmm... Honestly, for now, I wonder if it's better to just omit this.  I don't think, in its current form, the diagram adds much information.  I wonder if just describing this (similar to what you have) would be sufficient?

Extensions are built on web technologies such as HTML, JavaScript, and CSS.  They run in a separate,
sandboxed execution environment and interact with the Chrome browser.  Extensions are designed to "extend" the Chrome browser -- they utilize APIs in order to modify browser behavior and access web content.  There are two aspects to consider:

WDYT?

File: site/en/docs/extensions/mv3/overview/index.md [hunk 12]
```diff
@@ -6,30 +6,84 @@ updated: 2018-02-25
 description: An overview of the purpose of Chrome Extensions and how they're developed.
```
Comment (rdcronin) [inline site/en/docs/extensions/mv3/overview/index.md:6]: (random comment spot)

On the netlify preview, the link from https://deploy-preview-411--developer-chrome-com.netlify.app/docs/extensions/mv3/ to "what are extensions" is broken, but it looks fine in production.  Is that an issue with this PR, or with netlify?  Or where the link is technically wrong, but the production server catches and redirects?  (That's my closest guess?)

I can't see how this PR would have broken it, since it's not moving the file, but figured I'd raise it.

File: site/en/docs/extensions/mv3/overview/index.md [hunk 13]
```diff
@@ -6,30 +6,84 @@ updated: 2018-02-25
...
+code to access features of the browser itself: activating tabs, modifying net requests, and so on.
+
+To create an extension, you assemble some resources -- a manifest, JavaScript and HTML files,
+images, and others -- that constitute the extension. For development and testing, you can load these
+"unpacked" into Chrome using extension developer mode. Once you are happy with your extension, you
+can package it and distribute it to users.
+
+### How do users get extensions?
+
+Most Chrome users get extensions from the Chrome Web Store (CWS). Developers across the globe
```
Comment (rdcronin) [inline site/en/docs/extensions/mv3/overview/index.md:50]:
```suggestion
Most Chrome users get extensions from the Chrome Web Store. Developers across the globe
```

We don't use CWS anywhere else in this article, or generally in the docs AFAIK.  It's not really a popular abbreviation outside of Google, I think.  (Doing a google search for "CWS" or even "CWS extension" doesn't really show much)

**Review by rdcronin — APPROVED** [review]
Thanks, Andy!  This looks good once remaining nits are addressed, and we use a different example in lieu of PackageTrack (I'm fine deferring to Simeon for suggestions there, but ping me on chat if you want others).

File: site/en/docs/extensions/mv3/overview/index.md (hunk 12 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md:6]: Yeah, the link is wrong and redirecting in prod and on localhost but not in the preview. (Ran into this before where netlify doesn't do our redirects.) Fixed the links for good form.

File: site/en/docs/extensions/mv3/overview/index.md (hunk 11 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md:32]: Yeah, I wanted a diagram to a) help break up the slabs of text and b) help construct context. But I am fine with suppressing this for now and trying to come up with something more meaningful later.

**Review by dotproto — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/mv3/overview/index.md (hunk 9 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md:88]: Replaced with Google Similar Pages. I don't like the "preview unavailable" images, so I'm still shopping...

File: site/en/docs/extensions/mv3/overview/index.md (hunk 13 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md]: Done.

File: site/en/docs/extensions/mv3/overview/index.md [hunk 14]
```diff
@@ -109,28 +163,20 @@ or by pressing `Ctrl+Shift+F` on your keyboard.
 
 ## What next? {: #How-do-I-start }
```
[the remark this answers is not in the document]
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md:175]: Acknowledged.

File: site/en/docs/extensions/mv3/overview/index.md [hunk 15]
```diff
@@ -6,30 +6,84 @@ updated: 2018-02-25
 description: An overview of the purpose of Chrome Extensions and how they're developed.
 ---
 
-Extensions are small software programs that customize the browsing experience. They enable users to
-tailor Chrome functionality and behavior to individual needs or preferences. They are built on web
-technologies such as HTML, JavaScript, and CSS.
+This page provides a very brief introduction to what extensions are and how you can make a trivial one.
```
[the remark this answers is not in the document]
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md]: This kind of intro is standard tradecraft and eases the user into the page and its context. Doesn't matter that you could figure it out. Also appropriate that it's similar to, and lightly expands on, the description line: these appear in different places entirely. I do like the rewording for this bit of body text.

File: site/en/docs/extensions/mv3/overview/index.md [hunk 16]
```diff
@@ -6,30 +6,95 @@ updated: 2018-02-25
...
+
+Also shown in the diagram is an extension "plugged into" the Chrome browser. The extension can
+access and modify selected resources and actions in Chrome. There are two aspects to consider:
+{% endif %}
+
+Extensions let you "extend" the browser by using APIs to modify browser behavior and access web
+content. There are two aspects to consider:
+
+**The extensions user interface**&emsp;This provides a consistent way for users to manage their
+extensions.
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/overview/index.md:48]: I'm a little confused by this section. On one hand I feel like it's trying to describe elements that the extension developer needs to consider, on the other the actual text seems to be about browser level considerations.

What's the objective of this section? Asking in order to see if I have any ideas on how we might be able to rephrase this to better address that goal.

File: site/en/docs/extensions/mv3/overview/index.md [hunk 17]
```diff
@@ -6,30 +6,95 @@ updated: 2018-02-25
...
+diagram:
+
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/5qVlqGn6vVMCWCSRclOY.png", alt="Context diagram of Chrome with an extension", width="517", height="476" %}
+
+Also shown in the diagram is an extension "plugged into" the Chrome browser. The extension can
+access and modify selected resources and actions in Chrome. There are two aspects to consider:
+{% endif %}
+
+Extensions let you "extend" the browser by using APIs to modify browser behavior and access web
+content. There are two aspects to consider:
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/overview/index.md:45]: > There are two aspects to consider:

With regard to what?

File: site/en/docs/extensions/mv3/overview/index.md [hunk 18]
```diff
@@ -6,30 +6,95 @@ updated: 2018-02-25
...
+
+**The extensions user interface**&emsp;This provides a consistent way for users to manage their
+extensions.
+
+**Extensions APIs**&emsp;The [extensions APIs](/docs/extensions/reference/) allow the extension's
+code to access features of the browser itself: activating tabs, modifying net requests, and so on.
+
+To create an extension, you assemble some resources -- a manifest, JavaScript and HTML files,
+images, and others -- that constitute the extension. For development and testing, you can load these
+"unpacked" into Chrome using extension developer mode. Once you are happy with your extension, you
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/overview/index.md:55]: I think "developer mode" link to a related page. Perhaps https://developer.chrome.com/docs/extensions/mv2/tut_debugging/#debug_bg ?

File: site/en/docs/extensions/mv3/overview/index.md [hunk 19]
```diff
@@ -6,30 +6,95 @@ updated: 2018-02-25
...
+can package it and distribute it to users.
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/overview/index.md:56]: Perhaps we should link to packaging and distribution information, maybe https://developer.chrome.com/docs/webstore/publish/ ?

File: site/en/docs/extensions/mv3/overview/index.md [hunk 20]
```diff
@@ -6,30 +6,95 @@ updated: 2018-02-25
...
+
+### How do users get extensions? {: #getting-extensions }
+
+Most Chrome users get extensions from the Chrome Web Store. Developers across the globe
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/overview/index.md:60]: Make "Chrome Web Store" link to http://chrome.google.com/webstore

File: site/en/docs/extensions/mv3/overview/index.md [hunk 21]
```diff
@@ -6,30 +6,95 @@ updated: 2018-02-25
...
+publish their extensions in the Chrome Web Store where they are reviewed and made available to end
+users.
+
+{% Aside %}
+
+Some organizations use enterprise policies to install extensions on their user's devices. These
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/overview/index.md:66]: Make "enterprise policies" link to https://cloud.google.com/docs/chrome-enterprise/policies/

**Review by dotproto — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/mv3/overview/index.md (hunk 16 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md:48]: This is part of understanding what extensions are -- knowing the user experience is part of that. I'd like to get this merged, so can we steer away from optimization for now?

Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md:48]: I would prefer to gather further improvements in a follow-up ticket.

File: site/en/docs/extensions/mv3/overview/index.md (hunk 17 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md]: Rephrased this in 23fcebf

File: site/en/docs/extensions/mv3/overview/index.md (hunk 18 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md]: Done in 23fcebf

File: site/en/docs/extensions/mv3/overview/index.md (hunk 19 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md]: Done in 23fcebf

File: site/en/docs/extensions/mv3/overview/index.md (hunk 20 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md]: Done in 23fcebf

File: site/en/docs/extensions/mv3/overview/index.md (hunk 21 above)
Comment (awfuchs) [reply site/en/docs/extensions/mv3/overview/index.md]: Done in 23fcebf

File: site/en/docs/extensions/mv3/overview/index.md (hunk 16 above)
Comment (dotproto) [reply site/en/docs/extensions/mv3/overview/index.md:48]: That seems reasonable. I won't let good be the enemy of perfect here 😄

**Review by dotproto — APPROVED** [review]
I think we're in good shape. Thanks, @awfuchs!

**Review by HAVANA89 — COMMENTED** [review]
Hu
